axon-server-connector/src/main/java/org/axonframework/axonserver/connector/query/AxonServerQueryBus.java

```diff
@@ -164,6 +167,28 @@ public AxonServerQueryBus(Builder builder) {
         localSegmentAdapter = new LocalSegmentAdapter();
     }
 
+    @Override
+    public <Q, R> Publisher<QueryResponseMessage<R>> streamingQuery(StreamingQueryMessage<Q, R> query) {
+        int priority = priorityCalculator.determinePriority(query);
+        Scheduler scheduler = PriorityTaskSchedulers.forPriority(queryExecutor, priority, TASK_SEQUENCE);
+        return Mono.fromSupplier(this::registerStreamingQueryActivity)
+                   .flatMapMany(activity -> Mono.just(dispatchInterceptors.intercept(query))
+                                                .publishOn(scheduler)
```

Comment:
publishOn should be one of the last methods, since we want to push the query results to be provided to the callers on the scheduler.

Reply:
You are right, it should be just after deserialization. Let me fix it :)